### PR TITLE
adding bob sinkovits project

### PR DESCRIPTION
**Pull request by ChristinaLK**

### CODE REVIEW

File: projects/XSEDE_ECSS.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Name: XSEDE_ECSS 
+Description: Support for XSEDE users through ECSS
+Department: San Diego Supercomputing Center
+FieldOfScience: Computer Science
+Organization: University of California, San Diego
+PIName: Robert Sinkovits
+
+ID: '714'
```
Comment (brianhlin) [inline projects/XSEDE_ECSS.yaml:8]:
```suggestion
ID: '715'
```

There was another project PR sitting in our queue so we've got to update the ID. @matyasselmeci and I have been talking about what it'd take to remove this field so hopefully we can avoid this in the future!

**Comment by brianhlin** [comment]
Why the XSEDE prefix if the institution is UCSD?

**Comment by ChristinaLK** [comment]
He seemed to want the project associated w/ his role as the XSEDE ECSS lead. I could change it though, if that makes sense.

**Comment by brianhlin** [comment]
Makes sense to me, just wanted to check!

**Pull request merged**